gamemode/modules/base/sv_interface.lua

```diff
@@ -67,6 +67,12 @@ DarkRP.retrievePlayerData = DarkRP.stub{
             description = "The function that receives the information.",
             type = "function",
             optional = false
+        },
+        {
+            name = "failure",
+            description = "The function that is called when the information cannot be retrieved.",
+            type = "function",
+            optional = false
```

Comment:
of course, but I thought he was talking about the if statement, not the
failure callback

this English is 10/10 :+1

On Thu, Mar 17, 2016 at 1:35 PM, Falco Peijnenburg <notifications@github.com
> wrote:

> In gamemode/modules/base/sv_interface.lua
> <https://github.com/FPtje/DarkRP/pull/2368#discussion_r56503734>:
>
> > @@ -67,6 +67,12 @@ DarkRP.retrievePlayerData = DarkRP.stub{
> >              description = "The function that receives the information.",
> >              type = "function",
> >              optional = false
> > +        },
> > +        {
> > +            name = "failure",
> > +            description = "The function that is called when the information cannot be retrieved.",
> > +            type = "function",
> > +            optional = false
>
> You calling it a micro optimisation makes as little sense to me as calling
> it a check. It's a callback.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/FPtje/DarkRP/pull/2368/files/0dbb2b2959e7a8083d2cd74b89c97a5cb07bb4ba#r56503734>
>